deployment/charts/templates/fuel-core-deploy.yaml

```diff
@@ -138,6 +183,11 @@ spec:
           env:
             - name: HUMAN_LOGGING
               value: {{ .Values.app.human_logging | quote }}
+            - name: RESERVED_NODES
+              value: {{ .Values.app.reserved_nodes | quote }}
+            - name: BOOTSTRAP_NODES
+              value: {{ .Values.app.bootstrap_nodes | quote }}
+            # TODO: do we need to do anything to make this more optional for non-consensus nodes?
```

Comment:
@Voxelot Do we want to do something with this todo?=)

Reply:
I think we want to avoid mounting the consensus key secret to sentries (even if it's unused) for better security.

Reply:
we can address this later on when we are closer to a main-net hardened deployment config